README.md

```diff
@@ -43,9 +45,62 @@ $ omero zarr export 1 --bf --output /home/user/zarr_files
 Image exported to /home/user/zarr_files/2chZT.lsm
 ```
 
-To export masks for an image:
+To export masks for an Image:
+
+```
+# Saved under zarr_files/1.zarr/masks/0
+$ omero zarr masks Image:1 --output /home/user/zarr_files
+
+# Specify the mask-path (default 'masks').
+# e.g. Export to 1.zarr/my_masks:
+$ omero zarr masks Image:1 mask-path=my_masks
+
+# Specify the mask-name. (default is '0')
+# e.g. Export to 1.zarr/masks/A
+$ omero zarr masks Image:1 mask-name=A
+```
+
+The default behaviour is to export all masks on the Image to a single 5D
+"labelled" zarr array, with a different value for each mask Shape.
+An exception will be thrown if any of the masks overlap.
+
+To handle overlapping masks, you can export as a 6D zarr array or
+split one zarr group per ROI, using ROI ID as the group name.
+
+```
+# 6D mask saved under 1.zarr/masks/0
```

Comment:
IIUC we've decided against this and it's not compliant with the spec since it's 6D, should we remove it from the plugin?

Reply:
Certainly 6D mask wouldn't be handled well by `ome-zarr-py` napari plugin.

Reply:
@joshmoore Do we want to keep `6d` option? If so, I'll mark it as "doesn't comply with ome-zarr spec".

Reply:
I dropped it from the README. `-h` will warn about non-compliance.